src/content-feature.js

```diff
@@ -235,6 +235,33 @@ export default class ContentFeature {
     load (args) {
     }
 
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
+     * @type {(...args: any[]) => any}
+     */
+    notify (...args) {
+        const [name, params] = args
+        this.messaging.notify(name, params)
+    }
+
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
```

Comment:
I think explain that these are overloaded by specific types in the types/ dir. (we're gonna forget how this wires into the code in a week or so)